src/jquery.quicksearch.js

```diff
@@ -177,7 +184,12 @@
 			}
 			
 			if (noresults) {
-				this.results(false);
+				if(options.onNoResultsFound !== null){
+					options.onNoResultsFound();
+				}else{
+					this.results(false);
+				}
+				
```

Comment:
In my case, i needed to search others  results via ajax if there were no results in my table, but i didn't want to show the 'noResults' before run the onNoResultsFound function.

The 'this.results(false)' call this function: 

and that shows the 'noResults' message if the selector is defined.
`this.results = function (bool) {
			if (!!options.noResults.length) {
				options.noResults[bool ? 'hide' : 'show']();
			}
			
			return this;
		};
`
-